docs/guides/design.md

```diff
@@ -0,0 +1,281 @@
+---
+nav: Design guidance
+description: How to design your Pack so that it's easy to use and support.
+---
+
+# Design your Pack
+
+Coda Packs don't include traditional user interface elements like dialogs or sidebars. Instead users interact with your building blocks using standard Coda interfaces, like the formula editor. However there are still many subtle design choices to make when building your Pack, and they can have a real impact on usability.
+
+This page aims to provide the guidance you need to create a Pack that meets the needs and expectations of Coda users.
+
+
+## General guidance
+
+No matter what kind of Pack you are building, there are some basic rules to keep in mind.
+
+
+### Build building blocks
+
+Unlike other types of integrations, a Coda Pack doesn’t prescribe an exact end-to-end experience. Instead it provides a new set of building blocks, like formulas or buttons, that a user can deploy to improve their docs. These building blocks need to provide sufficient flexibility so that they can be combined in novel and bespoke ways.
```

Comment:
Might be worth also mentioning that users however do in fact benefit from seeing specific examples of your building blocks in action and key use cases already illustrated, and for that we have Featured Docs which are absolutely a best practice when it comes to shipping your pack.

Reply:
Great callout. I added a line at the end of this section, with a link out to Joe's guide on featured docs.